dali/test/python/test_audio_decoder_utils.py

```diff
@@ -24,3 +27,9 @@ def rosa_resample(input, in_rate, out_rate):
     ret[:,c] = a
 
   return ret
+
+def get_audio_files(fmt):
```

Comment:
Maybe it could be generalized to `get_files(path, fmt, ext)`
path - "audio"
fmt = ext in this case

Reply:
Done